game/screen_sing.cc

```diff
@@ -701,6 +702,7 @@ ScoreWindow::ScoreWindow(Instruments& instruments, Database& database):
 		if (item.track_simple == TrackName::DRUMS) item.color = Color(0.1, 0.1, 0.1);
 		else if (item.track_simple == TrackName::BASS) item.color = Color(0.5, 0.3, 0.1);
 		else item.color = Color(1.0, 0.0, 0.0);
+		item.player_id = item.track_simple; //probably not totally unique, but at least lets to discern some players with their favorite instrument
```

Comment:
The name of their instrument might indeed not be  unique.. maybe make an unique identifier for this and add a 'label' field or such to give their instrument a name ?

Reply:
@Baklap4 could you advice how to implement this unique id, please? :)

Reply:
@allcreater Can you ID band players and dancers by their controllers (which map to joysticks or keyboard), and singers by microphone number?